applications/StructuralMechanicsApplication/tests/formfinding_test/Fofi_4Point_Tent_noCable_test_materials.json

```diff
@@ -12,7 +12,7 @@
                             "KratosMultiphysics.POISSON_RATIO": 0.00000E+00,
                             "KratosMultiphysics.DENSITY": 1.00000E+000,
                             "KratosMultiphysics.THICKNESS": 1.00000E+000,
-                            "KratosMultiphysics.StructuralMechanicsApplication.MEMBRANE_PRESTRESS": [1.0, 1.0, 0.0]
+                            "KratosMultiphysics.StructuralMechanicsApplication.MEMBRANE_PRESTRESS": [[1.0, 1.0, 0.0]]
```

Comment:
This can now simply be `MEMBRANE_PRESTRESS`
Same for the other variables

Reply:
done